src/note.lisp

```diff
@@ -41,6 +41,11 @@ a CL:FORMAT control string referencing the values contained in ARGS."
     (record-note timestring " " outstring)
     outstring))
 
+(eval-when (:load-toplevel)
+  ;; hook, even if Actors isn't loaded...
+  ;; If Actors are loaded later, they will respect the hook
+  (setf (get :actors :print-handler) (um:curry 'note "~A")))
+
```

Comment:
Adding an override hook on Actors, just in case Actors has been loaded and uses its PR function anywhere.